gdtoolkit/linter/__init__.py

```diff
@@ -81,7 +81,9 @@
         "mixed-tabs-and-spaces": None,
         # misc
         "excluded_directories": {".git"},
-        # no-else-return
+        "no-else-return": {
+            "allow-elif": True
```

Comment:
please extract to `no-elif-return` rule in order to:

1. simplify in general (single responsibility principle)
2. allow disabling `no-else-return` without disabling `no-elif-return`

Reply:
I bundle them together as it is how ESLint seems to have done it. But they could be two separate rules too. I first thought that allowing `elif` could influence how to detect an `else` statement to remove. However, writing an example, I realised that it shouldn't. I need to fix that.

I want to keep them both in the same file, as they share logic. I could rename the file from `no_else_return_check.py` to `if_return_checks.py`.

I could also extract the common logic and create two separate files, but the methods are rather specific. What do you prefer?

Reply:
`if_return_checks.py` would be good enough :)